donor-portal/src/pages/newsroom/styles.css

```diff
@@ -42,6 +42,16 @@
   color: var(--elixir-color-red);
 }
 
+.news-room-image{
+  height: 600px;
+  background: url('./images/kandy_donation.jpg') no-repeat center;
```

Comment:
Let's import the image instead of this.  You can refer to this [file](https://github.com/LSFLK/MedicinesforLK/blob/development/donor-portal/src/pages/home/index.tsx) to see how to import and use them